src/test/resources/api_fixtures/billingportal/configuration.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
We shouldn't add this. stripe-mock should have the right fixtures instead so we should remove and fix the test above